src/interactive/plots.ts

```diff
@@ -134,21 +134,48 @@ export function plotPaneDelAll() {
     }
 }
 
+// wrap a source string with an <img> tag that shows the content
+// scaled to fit the plot pane unless the plot pane is bigger than the image
+function wrap_imagelike(srcstring: string) {
+    const html = `
+    <html style="padding:0;margin:0;">
+        <body style="padding:0;margin:0;">
+            <div style='width: 100%; height: 100vh'>
+                <img style='display:block; height: 100%; width: 100%; object-fit: scale-down; object-position: 0 0;' src='${srcstring}'>
+            </div>
+        </body>
+    </html>`
+    return html
+}
+
 export function displayPlot(params: { kind: string, data: string }) {
     const kind = params.kind
     const payload = params.data
 
     if (kind === 'image/svg+xml') {
-        g_currentPlotIndex = g_plots.push(payload) - 1
+        const has_xmlns_attribute = payload.includes("xmlns=");
```

Comment:
```suggestion
        const has_xmlns_attribute = payload.includes('xmlns=');
```
The check here isn't particularly robust, but that's fine I think.